content/docs/people/reading-material/working-student-perks.md

```diff
@@ -37,8 +37,8 @@ If you have any questions feel free to contact Team Up.
 
 🏝 paid holiday
 
-As a working student in Germany, you are entitled to paid vacation days, just like regular employees. The number of days you get depends on how many days per week you work, e.g. if you work 3 days per week: 12 vacation days per year. If you want to take more holiday or have further question feel free to ask Team-Up.
+As a working student in Germany, you are entitled to paid vacation days, just like regular employees. (20 days for Full Time Work). The number of days you get depends on how many days per week you work, e.g. if you work 3 days per week: 12 vacation days per year. (20/5*3) If you want to take more holiday or have further question feel free to ask Team-Up.
```

Comment:
Is the `(20/5*3)` formula intended in this way and in this place or a copy paste accident? Maybe describe it? :)